compiler/rustc_errors/src/lib.rs

```diff
@@ -657,6 +660,8 @@ impl Handler {
     /// * `can_emit_warnings` is `true`
     /// * `is_force_warn` was set in `DiagnosticId::Lint`
     #[rustc_lint_diagnostics]
+    #[allow(rustc::diagnostic_outside_of_impl)]
+    #[allow(rustc::untranslatable_diagnostic)]
```

Comment:
I'm surprised that this is necessary, maybe we should check for the old names and the new names of the diagnostic items in the lint and leave a comment with `cfg(bootstrap)` in it so that the old names are removed when the beta compiler is updated past this.

Reply:
These are not needed. Added them while testing some things in the experimental branch. 
Removed all of them [here](https://github.com/rust-lang/rust/pull/101558/commits/6cb12dff05f903ecc1d333c2ad1d1a779412b439).
Thank you!